<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bgwidget</class>
 <widget class="QDialog" name="bgwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget

#bgwidget { background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(92, 149, 255, 255), stop:1 rgba(255, 255, 255, 255))}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 48pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Delete Case?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_2_btn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>480</y>
     <width>311</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color: rgb(252, 233, 79);
font: 32pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="patientid">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="patientname">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>250</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Patient ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Patient Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="error">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>430</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="query_btn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>530</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color: rgb(233, 185, 110);
font: 24pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="patientemail">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>330</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Patient Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="caseid">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>90</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Case ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>421</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Checking DateTime</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="checkingdatetime">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>400</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>delete_2_btn</zorder>
  <zorder>patientid</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>error</zorder>
  <zorder>query_btn</zorder>
  <zorder>patientemail</zorder>
  <zorder>label_4</zorder>
  <zorder>patientname</zorder>
  <zorder>caseid</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>checkingdatetime</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
